tests/publiccloud/prepare_instance.pm

```diff
@@ -45,6 +45,7 @@ sub run {
 
     $instance->network_speed_test();
     $instance->check_cloudinit() if (is_cloudinit_supported);
+    $instance->enable_kdump();
```

Comment:
I think we should make it optional . Do think it is good idea that ALL tests will become red when something will break in kdump setup